src/AutoRest.CSharp/Common/Output/Expressions/Statements/UsingDeclareVariableStatement.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.IO;
```

Comment:
Do we need this change?

Reply:
Oh maybe we do not need it. I will clean it up